### DISCUSSION TITLE
ValidationException message translatable

### DISCUSSION THREAD

**Discussion by Banescu-Adrian** [laravel/framework#43066] [Ideas]
Greetings,
Can we have the option to translate the message from ValidationException ( The given data was invalid. ) from lang validation?

**Comment by henzeb** [comment]
Doesn't it do that already? https://laravel.com/docs/9.x/validation#specifying-custom-messages-in-language-files

**Reply by Banescu-Adrian** [reply]
The main message returned by FormRequest is declared in ValidationException class constructor. Can you override it from lang files?

**Reply by henzeb** [reply]
I see what you mean, but I am not sure how you got there. because a validation exception is thrown when there is at least one validation error. it's just a backup text in case there is no validation error.

**Reply by Banescu-Adrian** [reply]
Thank you for your reply,
Every 422 returns a message property and an errors property.
`{
"message":"The given data was invalid.",
"errors":{
    "name":["The name field is required."]
}}`
Maybe i haven't made myself clear enough. But i'm interested in the first property if it's available for translation :)

**Reply by henzeb** [reply]
Yes, I do understand, but I am puzzled as the summarize method that returns the name should use the first error instead. Although that does not fully translate when there are more errors (as it adds 'and x more errors')...

should be simple to implement though. you could create a PR for that.
